crates/matrix-sdk-ui/src/sync_service.rs

```diff
@@ -487,6 +496,14 @@ struct SyncServiceInner {
     /// The offline mode is described in the [`State::Offline`] enum variant.
     with_offline_mode: bool,
     state: SharedObservable<State>,
+
+    /// The parent tracing span to use for the tasks within this service.
+    ///
+    /// Normally this will be [`Span::none`], but it may be useful to assign a
+    /// defined span, for example if there is more than one active sync
+    /// service.
```

Comment:
Did you somewhere encounter the "more than one active sync service" case?

That should only happen in a multi-account scenario.

Reply:
Yeah, I was struggling with this in an integration test.